app/src/main/res/layout/topic_train_skill_view.xml

```diff
@@ -15,18 +15,28 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="wrap_content"
+    app:barrierDirection="right"
+    app:constraint_referenced_ids="button_example,text_view_status">
+
+    <androidx.constraintlayout.widget.Barrier
```

Comment:
Do you still need this barrier if you're using gravity:top in the checkbox?

Reply:
Yes, otherwise check icon is coming at the center_vertical of the text.